### PR TITLE
Add makefile-project-workspace-creator build system

### PR DESCRIPTION
**Pull request by tarc**
Specify library name and version:  **mpc (The Makefile, Project and Workspace Creator)/4.1.46**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by tarc** [comment]
This is a build requirement for #2040

File: recipes/makefile-project-workspace-creator/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("conan", "makefile-project-workspace-creator", "objectcomputing", "installer")
+    settings = "os"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def requirements(self):
+        if self.settings.os == "Windows":
```
Comment (madebr) [inline recipes/makefile-project-workspace-creator/all/conanfile.py:19]:
```suggestion
        if tools.os_info.is_windows:
```

Comment (madebr) [reply recipes/makefile-project-workspace-creator/all/conanfile.py]: @tarc Looking at it again, I think you were correct about using `self.settings.os`...
Sorry :see_no_evil: :smile:

**Comment by madebr** [comment]
@tarc The error is caused by my suggestion. Please undo it.
And change the test in test_package to only run when not cross building.

**Comment by tarc** [comment]
> @tarc The error is caused by my suggestion. Please undo it.
> And change the test in test_package to only run when not cross building.

Thank you. Actually `tools.os_info.is_windows` was working for me just fine on a Windows host...

**Pull request merged**